corehq/apps/reports/standard/deployments.py

```diff
@@ -582,19 +581,7 @@ class ApplicationErrorReport(GenericTabularReport, ProjectReport):
 
     @classmethod
     def show_in_navigation(cls, domain=None, project=None, user=None):
-        return cls.has_access(domain, user)
-
-    @classmethod
-    def has_access(cls, domain=None, user=None):
-        domain_access = (
-            domain_has_privilege(domain, privileges.APPLICATION_ERROR_REPORT)
-            if domain else False
-        )
-        user_access = (
-            user.is_superuser or user.is_dimagi
-            if user else False
-        )
-        return user and (domain_access or user_access)
+        return user and toggles.APPLICATION_ERROR_REPORT.enabled(user.username)
```

Comment:
Do we want to continue to support the report for superusers and dimagi staff accounts?

Reply:
Given that this report isn't usable on production, it might just be unnecessary UI bloat to have this report shown by default for superusers/staff. 

We could add in a conditional where if the environment isn't production we then show the report for superusers and staff, but given how little this report was used I think it should be fine to just keep it behind a FF for all users.